middleware/call.go

```diff
@@ -0,0 +1,39 @@
+package middleware
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/sirupsen/logrus"
+	"os/exec"
+)
+
+func Call(method string, params ...interface{}) (interface{}, error) {
+	var args []string
+	args = append(args, "call")
+	args = append(args, method)
+	for _, entry := range params {
+		sanitized, err := json.Marshal(entry)
+		if err != nil {
+			logrus.Errorf("Failed to marshal parameters for middleware: %s", err)
+			return nil, err
+		}
+		args = append(args, string(sanitized[:]))
+	}
+	out, err := exec.Command(middlewareClientPath, args...).Output()
+	if err != nil {
+		logrus.Errorf("Middleware call to %s failed: %s", method, err)
+		return nil, err
+	}
+	var sanitizedResult interface{}
+	// booleans are not json dumped right now by middleware client
+	if string(out) == "True\n" || string(out) == "False\n" {
```

Comment:
Same for `None` (but it should be converted to `null`, not `none`)

Reply:
```
❯ midclt call etc.generate docker
null
❯
```
We already get `null` for `None` response ?